app.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express');
+const Sequelize = require('sequelize');
+
+const app = express();
+
+const connection = new Sequelize({
+  database: 'ferretdb',
+  username: 'ferret',
+  password: null,
+  host: 'localhost',
+  dialect: 'postgres',
+  define: {
+    timestamps: false,
+  },
+});
+
+const User = connection.define('user', {
+  firstName: {
+    type: Sequelize.STRING,
+    field: 'first_name',
+  },
+  lastName: {
+    type: Sequelize.STRING,
+    field: 'last_name',
+  },
+  email: {
+    type: Sequelize.STRING,
+    field: 'email',
+    unique: 'true',
+    default: 'salutti',
+  },
+  password: {
+    type: Sequelize.STRING,
+    field: 'password',
+  },
+  rating: {
+    type: Sequelize.DOUBLE,
+    field: 'rating',
+    defaultValue: 0,
+  },
+  active: {
+    type: Sequelize.BOOLEAN,
+    field: 'active',
+    defaultValue: true,
+  },
+});
+
+connection
+  .sync({
+    logging: console.log,
+    force: true, //recreates the entries of the table for testing purposes
+  })
+  .then(() => {
+    console.log("Connection to database worked");
+  })
+  .catch(err => {
+    console.error("Connection to database failed", err);
+  });
+
+const port = process.env.PORT || 3000;
+
+app.post('/api/signup', (req, res) => {
```

Comment:
As you can imagine, we are going to create a file `app/routes/user.js` where we are going to add the path definitions for the user model `app.post('/api/users`, userController.createUser)`. Then we need a user controller `app/controllers/user.js`.
We will use REST style (google it 😄)for this API.